content/docs/contributing/policy.md

```diff
@@ -3,59 +3,109 @@ title: Feature Policy
 description: 'cert-manager contributing guide: Contribution Policy'
 ---
 
-We are open to feature requests and PRs implementing those. If you plan on contributing a new feature into cert-manager we recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
+We love to receive both feature requests and PRs which add stuff to cert-manager; the community is at the heart of what we do!
 
-# Features we will not allow
+If you're thinking of adding a feature, we recommend you read this doc to maximize the chances of your contribution getting the attention it deserves and hopefully to get it merged quickly!
 
-Certain features have been requested before, but have not been implemented properly or not able to be implemented without breaking the security model.
+We recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
 
-Below is a list of a few of these:
+## Feature Sizing: Getting Your Change Accepted
 
-## Vendoring Kubernetes related APIs outside of the `k8s.io/` namespace
+We evaluate new features and PRs based on their size; either they're small or large.
```

Comment:
Absolute nit: I guess it is also whether the change that they involve is 'significant' or not

Reply:
Yeah, that's worth saying explicitly, thanks!